docs/modules/ROOT/nav.adoc

```diff
@@ -0,0 +1,8 @@
+* xref:index.adoc[Overview]
```

Comment:
We can remove everything from the docs structure (it's only meant for technical non-user facing docs)